### DISCUSSION TITLE
Final Project

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#56] [Group Assignment]
Post your final group assignment here!

**Comment by sjeyakum** [comment]
**Newbury Park Student Athletes' H2Opinions**, Michelle & Sadhana

Repository: https://github.com/zhamic7/aa191a-su24
Mapplication: https://zhamic7.github.io/aa191a-su24/

**Comment by nesslee123** [comment]
UCLAccessibility Stories

Repository: https://nesslee123.github.io/disability-project/
Mapplication: https://nesslee123.github.io/disability-project/Project1/index.html

**Comment by Keiya-Wada** [comment]
Missing Japanese Narratives from JAMsj:

Group Repo: https://github.com/kimada62/JAMsjJAnarrative
Website: to be submitted later to professor

**Comment by jasminereyz** [comment]
Project Sankofa-Cation: 

Group Repo: https://github.com/jasminereyz/Post-Incarceration-Health/tree/main

Website : https://jasminereyz.github.io/Post-Incarceration-Health/index.html

**Comment by chrissanrow** [comment]
SEAing Queer:

Group Repo: https://github.com/chrissanrow/SEAing-Queer

Website: https://chrissanrow.github.io/SEAing-Queer/home/

**Comment by stinest** [comment]
Community on Campus:

repository: https://github.com/stinest/24su-191a-group4
website: https://stinest.github.io/24su-191a-group4/mapplication/index.html

**Comment by kianaptran** [comment]
Food is Fuel:

Group Repo: https://github.com/pwinnie2026/group-ate
Website: https://pwinnie2026.github.io/group-ate/index.html
